gitops/snapshot_test.go

```diff
@@ -126,8 +126,8 @@ var _ = Describe("Gitops functions for managing Snapshots", Ordered, func() {
 		Expect(meta.IsStatusConditionTrue(updatedSnapshot.Status.Conditions, gitops.AppStudioTestSuceededCondition)).To(BeFalse())
 	})
 
-	It("ensures the Snapshots status can be marked as invalid", func() {
-		gitops.SetSnapshotIntegrationStatusAsInvalid(hasSnapshot, "Test message")
+	It("ensures the Snapshots status can be marked as error", func() {
+		gitops.SetSnapshotIntegrationStatusAsError(hasSnapshot, "Test message")
 		Expect(hasSnapshot).NotTo(BeNil())
 		Expect(hasSnapshot.Status.Conditions).NotTo(BeNil())
 		Expect(meta.IsStatusConditionTrue(hasSnapshot.Status.Conditions, gitops.AppStudioIntegrationStatusCondition)).To(BeFalse())
```

Comment:
Should we test for the `AppStudioIntegrationStatusErrorOccured` condition here?

Reply:
Currently we just have AppStudioIntegrationStatusErrorOccured, it's used in the "reason" field.  There is no AppStudioIntegrationStatusErrorOccuredCondition.

Reply:
My bad, I missed that it was a reason and not a new condition. We can maybe check the condition reason to confirm that it's `ErrorOccured`

Reply:
Added the check for the condition reason. Thanks!

Reply:
Thanks, looks good!